src/vdaf/prio3.rs

```diff
@@ -369,19 +372,71 @@ where
         self.typ.verifier_len()
     }
 
+    fn num_proofs(&self) -> usize {
+        self.num_proofs.try_into().unwrap()
+    }
```

Comment:
Ahh, well `usize` might not be `u64` (it could be `u32`, say, on a 32-bit processor, right?). I assumed this would be fallable because the behavior depends somewhat on the target architecture. Happily I was wrong.

Reply:
Done, and I've also inlined this funciton.